src/neo/Ledger/ContractState.cs

```diff
@@ -13,29 +13,20 @@ namespace Neo.Ledger
     public class ContractState : ICloneable<ContractState>, ISerializable, IInteroperable
     {
         public int Id;
+        public ushort UpdateCounter;
+        public UInt160 ScriptHash;
```

Comment:
```suggestion
        public UInt160 Hash;
```

Now it is no longer suitable to be called `ScriptHash`.

Reply:
Done